### PR TITLE
Line editing layered_store

### PR DESCRIPTION
**Pull request by christinerose**
Although this file may no longer be relevant with the Tezos usage, it was in the repo, so I edited it along with the rest. 

In several instances, the "pack file" is referred to rather than `irmin-pack` file. Should this be `irmin-pack` each time? Perhaps would `pack` file be more accurate or is just > pack file < correct?

### CODE REVIEW

File: doc/irmin-pack/design/layered_store.md [hunk 1]
```diff
@@ -270,27 +270,27 @@ Tezos node. For this reason, we take the following steps:
...
 of the next object-store and next suffix. However, the control file
 will still point to the current object store and suffix. On restart,
-the main process opens the control file, at which point it knows which
+the main process opens the control file. At this point it knows which
 files are the "real" object-store and suffix (those listed in the
-control file), and can delete any other files it finds (which will be
+control file). The main process can delete any other files it finds (which will be
 the partial files the worker was creating).
 
 If the worker process has just terminated and a crash occurs, then the
```
Comment (christinerose) [inline doc/irmin-pack/design/layered_store.md:296]: Li 299 doesn't have a complete sentence. Could you clarify please?

File: doc/irmin-pack/design/layered_store.md [hunk 2]
```diff
@@ -301,13 +301,13 @@ disk or not. If it was, then the main process restarts, reads
 `control` (which was successfully renamed from `control.nnnn` ), and
```
Comment (christinerose) [inline doc/irmin-pack/design/layered_store.md:301]: Should li. 299 read: `control.nnnn` over the current `control` was recorded persistently on
disk or it wasn't. 

Would you please clarify this sentence? It doesn't make sense as it's written.

File: doc/irmin-pack/design/layered_store.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 
 This is a short document to describe the design of a garbage
-collection mechanism for irmin-pack (the variant of Irmin currently
+collection mechanism for `irmin-pack` (the variant of Irmin currently
 used by Tezos).
 
 **TL;DR**
 
-The irmin pack file stores objects. References from one object to
+The `irmin-pack` file stores objects. References from one object to
```
Comment (art-w) [inline doc/irmin-pack/design/layered_store.md:13]:
```suggestion
A pack file stores Irmin's objects. References from one object to
```
The name "pack file" describes a file which packs multiple objects (so `irmin-pack` got his name from that way of organizing the data on disk). The previous sentence was ambiguous as you noticed, but I'm afraid that "the `irmin-pack` file" reformulation would carry a different meaning than intended... I'm not sure if my suggestion is clearer, let me know if it's still confusing or if you think of a better way to phrase it!

File: doc/irmin-pack/design/layered_store.md (hunk 1 above)
Comment (art-w) [reply doc/irmin-pack/design/layered_store.md:296]: Hm there's an extra empty line 298 (which needs to be removed) but the sentence continues below. Does the following makes more sense?
> If the worker process terminates just before a crash occurs, then either the current `control` file contains the atomically renamed `control.nnnn` worker's file, or it has been left untouched by the worker. If it was successfully renamed, then the main process will start operating on the new object-store and suffix file created by the worker (and will delete any residual old version of those files).

**Review by art-w — CHANGES_REQUESTED** [review]

File: doc/irmin-pack/design/layered_store.md (hunk 2 above)
Comment (art-w) [reply doc/irmin-pack/design/layered_store.md:301]: (marked as resolved because it's easier to discuss in your previous comment that contains the rest of the sentence)

File: doc/irmin-pack/design/layered_store.md (hunk 1 above)
Comment (christinerose) [reply doc/irmin-pack/design/layered_store.md:296]: Thank you! I fixed it and (hopefully) clarified two of those sentences. Please ensure they're still accuarte.

File: doc/irmin-pack/design/layered_store.md (hunk 3 above)
Comment (christinerose) [reply doc/irmin-pack/design/layered_store.md:13]: Perfect!
